js/script.js

```diff
@@ -0,0 +1,63 @@
+const bookTitleInput = document.querySelector('#form-title');
+const bookDescInput = document.querySelector('#form-desc');
+// const bookAddInput = document.querySelector('#form-add');
```

Comment:
- [OPTIONAL] You did a good job using JavaScript to get all the information from the HTML :+1:, but removing all the commented-out code would be better because it causes confusion and is not a good practice. You can check more about that in this [link](https://methodpoet.com/commented-out-code/#:~:text=Leaving%20commented%2Dout%20code%20in,or%20experimenting%20with%20the%20code.)